### ISSUE TITLE
invalid domains on telemetry_domains_beta.txt

### ISSUE THREAD

**Issue by kishansundar** [MoralCode/pihole-antitelemetry#13]
```
app−measurement.com
growth−pa.googleapis.com
iphone−ld.apple.com
mobilenetworkscoring−pa.googleapis.com
proﬁle.gc.apple.com
```

**Comment by MoralCode** [comment]
thanks for creating an issue! what do you mean by "invalid" in this case?

**Comment by kishansundar** [comment]
all the domains are Internationalizing Domain Names. special characters in the doamins
for e.g. iphone−ld.apple.com redirects to https://xn--iphoneld-cf7d.apple.com/

**Comment by MoralCode** [comment]
ah. it looks like this may be related to #11. 

Essentially most of the domains were pulled directly from the PDF with a script and likely either preserve the same formatting or had their formatting messed up.

Changing the dash symbol `iphone−ld.apple.com` to a single regular hyphen `-` the browser gives an access denied error rather than a "failed to load/connect" error, suggesting that its likely just a formatting error, however i haven't yet changed the domains  yet because it seems like it might be possible to use this as a strategy to avoid being blocked (i.e. use non-ASCII characters so blocklists dont exactly match the domain).

It also seems like googles video-serving domains for youtube may use a similar format (example: `https://r4---sn-nx5s7nee.googlevideo.com` appears to be encoded similarly to `https://xn--iphoneld-cf7d.apple.com/` since they are both two characters followed by some dashes, a seemingly arbitrary name, another dash, then some random looking alphanumeric charcaters) 

I will add the ASCII versions of these domains to the list as well, however i would like to learn more about the use of these internationalized domains before assuming that they are safe and removing them from the list

**Comment by danifr** [comment]
Same issue here:
```
  [i] Target: https://raw.githubusercontent.com/MoralCode/pihole-antitelemetry/main/telemetry-domains_beta.txt
  [✓] Status: Retrieval successful
  [i] Analyzed 24 domains, 6 domains invalid!
      Sample of invalid domains:
      - app−measurement.com
      - growth−pa.googleapis.com
      - iphone−ld.apple.com
      - mobilenetworkscoring−pa.googleapis.com
      - proﬁle.gc.apple.com
```

**Comment by MoralCode** [comment]
Thanks for the tip! i'll look into this and see if it solves the issue outlined above

edit, seems as though [pihole](https://github.com/pi-hole/pi-hole/blob/4c39edbeb91c04b83d4d78e4eaf669c3a8c04fc6/gravity.sh#L511) and [stackoverflow](https://stackoverflow.com/a/3523260) agree that these domains are invalid

**Comment by MoralCode** [comment]
fixed in e66c0482af152df86f63780421b4d700476faccb
